src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -1460,45 +1415,7 @@ class DistributedClosestPoint
   }
 
   /// Set the runtime execution policy for the query
-  void setRuntimePolicy(RuntimePolicy policy)
-  {
-    SLIC_ASSERT_MSG(
-      isValidRuntimePolicy(policy),
-      fmt::format("Policy '{}' is not a valid runtime policy", policy));
-    m_runtimePolicy = policy;
-  }
-
-  /// Predicate to determine if a given \a RuntimePolicy is valid for this configuration
-  bool isValidRuntimePolicy(RuntimePolicy policy) const
```

Comment:
No need to check for validity anymore because `RuntimePolicy` can only have valid values.